app/src/ui/toolbar/branch-dropdown.tsx

```diff
@@ -128,23 +130,24 @@ export class BranchDropdown extends React.Component<
 
       const pullRequestsWithStatus: Array<IPullRequest> = []
       for (const pr of pullRequests) {
```

Comment:
Just a note here that it'd be real nice if we could parallelize the fetching of status a bit, maybe run 5 or so requests in parallel but I imagine that requires some more sophisticated queuing work and running them all in parallel seems excessive.

Reply:
Yeah and this is all just placeholder so we can present some real data.